flow/raster_cache.cc

```diff
@@ -419,37 +400,85 @@ void RasterCache::PrepareNewFrame() {
   display_list_cached_this_frame_ = 0;
 }
 
+void RasterCache::SweepOneCacheAfterFrame(RasterCacheKey::Map<Entry>& cache,
+                                          RasterCacheMetrics& picture_metrics,
+                                          RasterCacheMetrics& layer_metrics) {
+  std::vector<RasterCacheKey::Map<Entry>::iterator> dead;
+
+  for (auto it = cache.begin(); it != cache.end(); ++it) {
+    RasterCacheKeyType type = it->first.type();
+    Entry& entry = it->second;
+
+    if (!entry.used_this_frame) {
+      dead.push_back(it);
+    } else if (entry.image) {
+      if (type == RasterCacheKeyType::kPicture ||
```

Comment:
A switch would ensure that all key types are dealt with.

Reply:
Done